.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+<!-- summary of changes (add screenshots/videos to the bottom of the PR if applicable)
+- relevant context from Slack is summarized and included -->
```

Comment:
This was a voice call, but it was pointed out that https://github.com/PrairieLearn/PrairieLearn/pull/12714 was a particularly bad example of something that was missing:

- Slack context/decisions
- Testing instructions
- Design decisions